client/src/views/ResetPwd.vue

```diff
@@ -0,0 +1,363 @@
+<template>
+    <body>
+        <div class="main-container" :class="{ 'mobile-container': mobile }">
+            <div class="main-container-top">
+                <h2 class="reset-pwd-title-txt" :class="{ 'mobile-title': mobile }">Reset Password</h2>
+                <div class="filler01"></div>
+                <img src="@/assets/logo.png" class="logo">
+            </div>
+            <div class="main-container-bottom">
+                <div class="pwd-container">
+                    <img src="@/assets/key.png" class="img-style1">
+                    <input type="password" class="input-style1" placeholder="Password" v-model="newPassword">
+                </div>
+                <div class="pwd-container">
+                    <img src="@/assets/key.png" class="img-style1">
+                    <input type="password" class="input-style1" placeholder="Confirm password" v-model="confirmPassword">
+                </div>
+                <button @click="changePwd" class="send-btn">CONFIRM</button>
+            </div>
+        </div>
+    </body>
+</template>
+
+<script>
+export default {
+    name: 'ResetPwd',
+    data() {
+        return {
+            windowWidth: false,
+            mobile: false,
+            email: '',
+            newPassword: '',
+            confirmPassword: '',
+        }
+    },
+    mounted() {
+        window.addEventListener('resize', this.checkScreen);
+        this.checkScreen();
+    },
+    methods: {
+        checkScreen() {
+            this.windowWidth = window.innerWidth;
+            if (this.windowWidth <= 960) {
+                this.mobile = true;
+            } else {
+                this.mobile = false;
+            }
+        },
+
+        async changePwd() {
+            if (this.newPassword !== this.confirmPassword) {
+                // Display an error message
```

Comment:
There is this display of an error message to do still, you can check other files like the Login to see what has already been done